### ISSUE TITLE
Multiple profiles

### ISSUE THREAD

**Issue by GlaireDaggers** [GlaireDaggers/ContentPipe#2]
Would be nice to support the concept of "profiles", which would be different build targets. For example, "PC" vs "Switch".
I think the general idea is that the profile would be set via commandline (if not specified, "Default" or something would be used). Build processors would get access to the current profile being built, and potentially could modify the JSON meta format to support per-profile settings like:

```json
{
    "Default": {}, // data here
    "PC": {}, // data here
    "Switch": {} // data here
}
```

**Comment by GlaireDaggers** [comment]
Might be worth waiting on https://github.com/GlaireDaggers/ContentPipe/pull/4 since it adds a BuildOptions parameter that would be a perfect place to add current build profile.

**Comment by GlaireDaggers** [comment]
Implemented as of fc36bafd3ee87378bdff18b3381093eed26e4fca
